inlong-manager/manager-dao/src/main/resources/mappers/StreamSourceEntityMapper.xml

```diff
@@ -120,6 +120,19 @@
             </if>
         </where>
     </select>
+    <select id="selectDataAddTaskCount" resultType="java.lang.Integer">
+        select count(1)
+        from stream_source
+        <where>
+            <if test="groupId != null and groupId != ''">
+                and inlong_group_id = #{groupId, jdbcType=VARCHAR}
+            </if>
+            <if test="streamId != null and streamId != ''">
+                and inlong_stream_id = #{streamId, jdbcType=VARCHAR}
+            </if>
+            and task_map_id is not NULL
```

Comment:
and is_deleted=0

Reply:
The purpose here is to obtain the quantity of all supplementary recording tasks under the stream, regardless of whether they have been deleted or not.